build-system/externs/amp.extern.js

```diff
@@ -476,9 +509,15 @@ AmpElement.prototype.togglePlaceholder = function (show) {};
 /** @return {{width: number, height: number}} */
 AmpElement.prototype.getLayoutSize = function () {};
 
+/**
+ * TODO: remove this when typechecking is restored to AMP.BaseElement.
+ * @typedef {*}
+ */
+var BaseElement;
```

Comment:
We should probably try standardize on `let`, at least for new entries

Reply:
Updating all to `var`